src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+import i18n from '../i18n'
+
+const sumAll = numbers => {
+  let points = 0
+  numbers.forEach(number => {
+    points = points + number
+  })
+  return points
+}
+
+export const isThreeOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  if (isFullHouse(numbers)) return true
+  let sameNumbers = 0
+  const foundSameNumbers = []
+  sortedArray.reduce((a, b) => {
+    if (a === b) {
+      foundSameNumbers.push(b)
+      sameNumbers = sameNumbers + 1
+    }
+    return b
+  })
+  return sameNumbers === 2 && foundSameNumbers[0] === foundSameNumbers[1]
+}
+
+const threeOfKindPoints = numbers => {
+  return isThreeOfKind(numbers) ? sumAll(numbers) : 0
+}
+
+export const isFourOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5 || isFullHouse(numbers)) return false
+  let sameNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a === b) sameNumbers = sameNumbers + 1
+    return b
+  })
+  return sameNumbers === 3
+}
```

Comment:
Yes, you have to return false because a Full House leads to `sameNumbers === 3`.
This need to check against Full House seems to be a huge downside on the choosen algorithm.

Reply:
`expect (isFourOfKind([1, 1, 1, 1, 1])).toBe(true)`